### DISCUSSION TITLE
local module imports fail after os.chdir

### DISCUSSION THREAD

**Discussion by raphCode** [inducer/pudb#556] [Troubleshooting]
**Describe the bug**

After `os.chdir()` local modules cannot be imported when run under pudb and raise `ModuleNotFoundError`.

**To Reproduce**

- create files:
  - `some_module.py` (empty)
  - `some_folder/` (empty directory)
  - `test.py` content as follows:

```python
import os

os.chdir("some_folder")
import some_module
```
- run `pudb3 test.py`
- import of `some_module` fails

**Expected behavior**

Module is imported successfully, no `ModuleNotFoundError` is raised.

**Additional context**

When run under the normal python interpreter this works.
The reason seems to be that `sys.path` is normally populated with an absolute path to the directory from which python is run.
When run under pudb, it is only populated with the empty string which does refer to the current directory, which is wrong after the `os.chdir()`.

**Versions**

Python 3.10.7
pudb:pudb3 v2022.1.2

**Comment by raphCode** [comment]
May I ask why this was moved to a discussion? Is it not considered a pudb bug?

In my use case I use the [hydra](https://hydra.cc/) framework to import functions from user-specified modules. This happens at runtime in my code, after hydra has changed to a new working directory.

Under pudb a previously working configuration now fails with `ModuleNotFoundError`.
I already have a workaround in my code, but I feel it should not be necessary just to make it work with a single specific debugger.

**Reply by inducer** [reply]
You're right, I didn't read carefully enough. Sorry about that. See #557.
